index.bs

```diff
@@ -522,6 +524,168 @@ See also:
 * [Security and privacy are essential](https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy)
 * [What data does this specification expose to an origin?](https://www.w3.org/TR/security-privacy-questionnaire/#underlying-platform-data)
 
+<h3 id="support-non-fully-active">Support non-"fully active" documents</h3>
+
+After a user navigated away from a document,
+the document might stay around in a non-[=Document/fully active=] state,
+and might be reused when the user navigates back to the entry holding the document.
+In the past, many APIs have missed specifying support for non-[=Document/fully active=] documents,
+making them hard to support in various user agents' implementation of "back/forward cache" (BFCache).
+
+To avoid this happening with your API,
+you should specify support for non-[=Document/fully active=] documents by following these guidelines:
+
+<h4 id="gate-fully-active">Gate actions with [=Document/fully active=] checks</h4>
+
+When performing actions that might update the state of a document,
+be aware that the document might not be [=Document/fully active=]
+and is considered as "non-existent" from the user's perspective.
+This means they should not receive updates or perform actions.
+
+Note: It is possible for a [=Document/fully active=] document to be perceived as "non-existent" by users,
+such as when the document is [displaying prerendered content](https://jeremyroman.github.io/alternate-loading-modes/).
```

Comment:
Hi - is there a more authoritative reference for this concept that could be linked here – e.g. something in standards track?

Reply:
cc @domenic @jeremyroman

Reply:
I don't think there is: prerendering is not well defined anywhere, despite being implemented in Safari stable and behind a flag in Chrome, so that is why we're trying to create a spec that defines it well.

Reply:
I wonder if we should instead point to https://html.spec.whatwg.org/#link-type-prerender ? Given that what's at the link above is supposed to replace what's in HTML anyway?

Reply:
https://html.spec.whatwg.org/#link-type-prerender doesn't really explain too much but I guess that's the most official link we can get for now. Changed it to that, but I'm fine with the old link as well.

Reply:
I'd strongly prefer the old link as the HTML section is so vague as to be useless; it should honestly be removed from HTML.

Reply:
hehe, I have to agree. I guess we can always update this document once HTML gets updated with something better.

Reply:
OK, reverted to old link then :)